Details about Debugging permissions (#96705)
Azure/azure-content: articles/synapse-analytics/security/synapse-workspace-understand-what-role-you-need.md

You can review logs and cancel running jobs and pipelines if you're a Synapse Compute Operator at the workspace or for a specific Spark pool or pipeline.  

### Debug pipelines

> You can review and make changes in pipelines with Synapse User Role but if you want to be able to debug it you also need Synapse Credential User

Reviewer: ```suggestion
You can review and make changes in pipelines as a Synapse User, but if you want to be able to debug it you also need to have Synapse Credential User.
```
